dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops

```diff
@@ -247,30 +246,36 @@ KRB5_LOGIN_FAILED_DEFAULT_KEYTAB=CWIML4510E: Kerberos login failed with the {0}
 
-CANNOT_READ_KRB5_FILE=CWIML4513E: The {0} LdapRegistry cannot read the {1} Kerberos file.
+CANNOT_READ_KRB5_FILE=CWIML4513E: The {0} LDAP registry cannot read the {1} Kerberos file.
 CANNOT_READ_KRB5_FILE.explanation=The Kerberos file cannot be opened. Either the file permissions are incorrect, or the file does not exist.
 CANNOT_READ_KRB5_FILE.useraction=Verify that the file location is correct and that the server has read file permissions.
 
-KRB5_PRINCIPAL_REQUIRED=CWIML4514E: The bind authentication mechanism on the {0} LdapRegistry is set to GSSAPI, but the required attribute, krb5Principal, is not configured.
-KRB5_PRINCIPAL_REQUIRED.explanation=The Kerberos principal name is required when GSSAPI (Kerberos) is enabled.
-KRB5_PRINCIPAL_REQUIRED.useraction=Add a krb5Principal attribute to the krbAuthentication elemenent on the LdapRegistry element.
+# CWIML4514 already used
 
 KRB5_FILE_NOT_FOUND=CWIML4515E: The {0} attribute from the {1} element is configured to a file that does not exist at the following location: {2}
 KRB5_FILE_NOT_FOUND.explanation=The configuration refers to a file that is either unreadable or does not exist.
 KRB5_FILE_NOT_FOUND.useraction=Ensure that the configuration points to a file that exists and is readable by the application process.
 
-KRB5_FILE_FOUND=CWIML4516E: The LdapRegistry component is configured to use a {0} file that is located at {1}.
+KRB5_FILE_FOUND=CWIML4516E: The LDAP registry component is configured to use a {0} file that is located at {1}.
 KRB5_FILE_FOUND.explanation=The configured file was successfully located.
 KRB5_FILE_FOUND.useraction=No action is required.
 
-IDMAP_INVALID_FORMAT=CWIML4518W: The {0} {1} value is malformed. The value must be a series of objectclass:attribute or *:attribute pairs, where each pair is separated by a semi-colon.
+# CWIML4517 already used
+
+IDMAP_INVALID_FORMAT=CWIML4518W: The {0} {1} value is malformed. The value must be a semi-colon separated list of 'objectclass:attribute' or '*:attribute' pairs.
 IDMAP_INVALID_FORMAT.explanation=The userIdMap value must be in a valid format so that the objectclass and attribute can be parsed.
 IDMAP_INVALID_FORMAT.useraction=Update the userIdMap value so that it adheres to the specified format.
+
+# CWIML4519 and CWIML4520 already used
+
+KRB5_SERVICE_NOT_AVAILABLE=CWIML4521E: The {0} LDAP registry attempted to bind to the LDAP server by using Kerberos credentials for the {1} principal name, but the Kerberos service that provides these credentials is not available. The bind authentication mechanism is {2}.
+KRB5_SERVICE_NOT_AVAILABLE.explanation=The Kerberos service was not available when the LDAP registry attempted to bind to the ldap server. If the Kerberos service is not available, no users can authenticate.
+KRB5_SERVICE_NOT_AVAILABLE.useraction=Review the LDAP registry configuration settings for bindAuthMechanism, krb5Principal and krb5TicketCache attributes as well as the Kerberos configuration. Restart the server.
```

Comment:
Add "the" before bindAuthMechanism

Review the LDAP registry configuration settings for bindAuthMechanism, krb5Principal and krb5TicketCache attributes...
-->
Review the LDAP registry configuration settings for the bindAuthMechanism, krb5Principal and krb5TicketCache attributes